paradise_papers_search/fetch_api/views.py

```diff
@@ -1,7 +1,7 @@
 from rest_framework.views import APIView
```

Comment:
I see we are using this rest_framework... are we sure we need it? I mean if there is a way to do anything with Django we should use Django, if not this is okay.

Reply:
The framework is necessary for the type of responses we are using and it is specially designed for Django